### ISSUE TITLE
Style/SafeNavigation: Use safe navigation (&.) - not applicable

### ISSUE THREAD

**Issue by Rataah** [bbatsov/rubocop#4771]
Rubocop raises the error: **Style/SafeNavigation: Use safe navigation (&.)** 
for the following statement:

```ruby
process_file(file_object.file_content) if file_object && file_object.status != 'blank'
````

The behavior expected by Rubocop (see Actual behavior) change the behavior of the line of code.
If `file_object=nil`, the statement `file_object&.status != 'blank'` will be true

--------

## Expected behavior

We expect Rubocop to allow this case

## Actual behavior

Style/SafeNavigation: Use safe navigation (&.)

Rubocop expect our code to be:
```ruby
process_file(file_object.file_content) if file_object&.status != 'blank'
````

## RuboCop version

```
$ rubocop -V
0.50.0 (using Parser 2.4.0.0, running on ruby 2.4.2 x64-mingw32)
````

**Comment by mikegee** [comment]
This issue appears to be a dupe of https://github.com/bbatsov/rubocop/issues/4769.

**Comment by biow0lf** [comment]
One more example:

```ruby
if user && user.retries >= 3
  # code
end
```

**Comment by Drenmi** [comment]
Fixed in #4749.
